Tech Stack Section Added

abhi03ruchi opened the pull request:
## Related Issue

Closes: #138 

## Description of Changes
I have done changes in readme section of the file.

For example:
- Added a new function to handle XYZ.
- Updated the ABC module to fix the bug.
- Refactored code in the DEF class for improved performance.
-->

## Checklist:

<!--
Mark the checkboxes to indicate completion. Example:
- [x] My code follows the style guidelines of this project.
-->

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [ ] I have included comments in areas that may be difficult to understand.
- [x] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

## Screenshots

|           Original           |           Updated           |
| :--------------------------: | :-------------------------: |
| ![Original screenshot](link) | <img width="392" alt="Screenshot 2023-05-29 at 10 41 21 PM" src="https://github.com/anmode/grabtern-frontend/assets/105661636/4377aaa3-aba9-48ca-bf89-73d04ad2a6bf">

Please provide any necessary screenshots to illustrate the changes made.

anmode commented:
Please add the contributor list and make some techicality. How to setup project in detail? and what is prettier lint test and how to fix it?

abhi03ruchi commented:
I have updated the readme 

- added  contributor list 
- badges
- How to setup project in detail?(Contributing) 
- what is prettier lint test and how to fix it?


https://github.com/anmode/grabtern-frontend/assets/105661636/be67cc08-10e7-46b7-bd15-269cf4109752

anmode commented on README.md:
````diff
@@ -22,11 +55,79 @@ npm run dev
+<b>"Prettier lint test" </b> refers to running a linter (such as ESLint) with the Prettier plugin or rules enabled. It means checking your code for both linting errors and code formatting inconsistencies according to Prettier's rules.
+
+To fix Prettier linting issues, you can follow these steps:
+
+1) Install necessary tools: Ensure that you have the required tools installed in your project. You'll need ESLint and Prettier, along with any additional plugins or configurations specific to your project or coding environment.
+```bash
+npm install eslint prettier --save-dev
+```
+2) Configure ESLint: Set up your ESLint configuration file (.eslintrc.js or .eslintrc.json) to include Prettier rules and the Prettier plugin. This allows ESLint to use Prettier's formatting rules for linting.
+```bash
+module.exports = {
+  // Other ESLint rules and configurations
+  extends: ['eslint:recommended', 'plugin:prettier/recommended'],
+};
+```
+Make sure the Prettier plugin is included in the extends array.
+
+3) Configure Prettier: Create a Prettier configuration file (.prettierrc or prettier.config.js) in the root of your project. Define your preferred formatting rules and options for Prettier.
+``` bash
+module.exports = {
+  // Prettier options
+};
+```
+Customize the Prettier options according to your desired code formatting style.
+
+<br>
````
Actually this is wrong... 
1. You can include to have prettier extension installed in your vs code. so when you save the code it will formated automatically
2. If you do not want to install prettier there are two command that is helpful
  1. `npx prettier --check .` - It will check where are formatting prblm
  2. `npx prettier --write .` - it will fix the formatting

abhi03ruchi replied:
Plz review :- 
<img width="1043" alt="Screenshot 2023-05-31 at 10 25 27 AM" src="https://github.com/anmode/grabtern-frontend/assets/105661636/91317262-937a-48ea-a3b5-1d57d1ad868e">
<img width="1043" alt="Screenshot 2023-05-31 at 10 26 09 AM" src="https://github.com/anmode/grabtern-frontend/assets/105661636/5ad559d2-7b68-4bbe-bedc-db85e43c6fd4">

anmode reviewed (changes requested):
One more thing you did not include how to setup project. please include it...if you find any problem I'm here to help thanks

anmode reviewed (approved):
Great! @abhi03ruchi.
